application.py

```diff
@@ -34,6 +34,8 @@
 from sqlalchemy import create_engine
 engine = create_engine(repo_db_conn)
 kr_names  = engine.table_names()
+engine.dispose()
```

Comment:
@abhirathb What does this do?

Reply:
close the connection. I have been getting "too many connections" error from the DB